templates/vars/aks/prod.json

```diff
@@ -3,22 +3,22 @@
   "contentVersion": "1.0.0.0",
   "parameters": {
     "kubernetesVersion": {
-      "value": "1.17.9"
+      "value": "1.18.8"
     },
     "nodeMinCount": {
       "value": 32
     },
     "nodeMaxCount": {
-      "value": ""
+      "value": "40"
     },
     "enableAutoScaling": {
-      "value": ""
+      "value": "true"
     },
     "type": {
-      "value": "AvailabilitySet"
+      "value": "VirtualMachineScaleSets"
     },
     "vmSize": {
-      "value": "Standard_D4s_v3"
```

Comment:
same here why are we changing the vmsize?

Reply:
yeah it's corrected now